pkg/query-service/model/v3/v3.go

```diff
@@ -112,6 +112,22 @@ func (a AggregateOperator) RequireAttribute(dataSource DataSource) bool {
 		default:
 			return true
 		}
+	case DataSourceLogs:
+		switch a {
+		case AggregateOperatorNoOp,
+			AggregateOperatorCount:
+			return false
+		default:
+			return true
+		}
+	case DataSourceTraces:
+		switch a {
+		case AggregateOperatorNoOp,
+			AggregateOperatorCount:
+			return false
+		default:
+			return true
+		}
```

Comment:
No, we can add that as well

Reply:
> Does it form a valid SQL query when there is no aggregate attribute because I believe count(%s) is used.

Yes it uses `count()`. Here's an example query:
`SELECT toStartOfInterval(timestamp, INTERVAL 60 SECOND) AS ts, count()/60 as value from signoz_traces.distributed_signoz_index_v2 where (timestamp >= '1680066360726210000' AND timestamp <= '1680066458000000000') group by ts order by ts`

> No, everything is metrics that require an aggregate attribute i.e. metric name. I am not sure why the metrics case existed above this change. Can you please remove the whole case for metrics?

Sure

Reply:
After removing all cases for metrics. When a user opens by dashboard query builder, by default user will see below error which is not a good user experience:
<img width="885" alt="Screenshot 2023-05-17 at 21 06 15" src="https://github.com/SigNoz/signoz/assets/9512100/20631036-74c6-4003-8229-a3904f125bb6">

Reply:
To solve this bad user experience frontend change is required. This will be picked later. So allowing empty aggregate attribute for `NoOp` in metrics for now.
cc: @srikanthccv